content/docs/guide-development.md

```diff
@@ -117,7 +117,51 @@ https://renkulab.io/
 
 TODO: explain how to run server
 
+To support the development of workflows in Renku, a set of dedicated funcitonailities, provided as Renku plugins, are made available. In particular those aim to:
 
+* Offer an interactive (and non) visualization of the project Knowledge Graph (`renku-graph-vis` plugin)
+* Create dedicated annotations for the execution of `astroquery` functions and add those to the project Knowledge Graph (`renku-aqs-annotation` plugin)
+
+Those two set of features are provided within different plugins.
+
+### Visualizing project Knowledge Graph with `renku-graph-vis` plugin
+
+Starting from the knowledge graph, this is used to generate an interactive graphical representation of the graph itself.
+
+The plugin provides two CLI commands:
+  * `display` to generate a representation of the graph over an output image
+  * `show-graph` to start an interactive visualization of the graph over the browser
+
+In addition, the plugin offers a dynamic visualization of the graph during an interactive renku session.
+
+The primary benefit introduced is the ability to have a live overview of the ongoing development, during an interactive Renku session. This can be seen within the animation below, where the graph is automatically updated with information about the execution of a notebook upon its completion.
+
+![](renkulab_execution_example_2.gif)
+
+Additionally, an ontology can be integrated into the graph, providing valuable insights into the types of entities inside it. In the image below, it can be seen that the `SimbadClass` node is an instance of the `AstroqueryModule` class, while `Mrk 421` is an instance of the `AstrophysicalObject` class.
```

Comment:
I wanted to explain that an ontology can be imported. I will further explain that in the image the ODA ontology has been used.

Reply:
Yes but ontology is already integrated in the graph, it's essential part of it, even just by using it's entities.
What is done is that ontology is shown.

Reply:
ok then, I was more writing it using an approach oriented to the development, which is not the scope of this document. am I right?

Reply:
Scope of this document  is guide for workflow development. Looking on the ontology helps with workflow development since it suggests how is the workflow related to known terms of the ontology, and hence how this workflow can be used and integrated.

Reply:
ok, I integrated also this about the ointology

Reply:
anything else missing?